README.md

```diff
@@ -33,15 +33,19 @@ Allows for dividing tenants into different AWS accounts.  [byAccount](docs/byAcc
 
 To use, pass your `AmazonDynamoDBClientBuilder`, `MtAccountCredentialsMapper`, and `MtAmazonDynamoDbContextProvider` to the builder.  At runtime, a String representing the tenant identifier will be passed to your credentials mapper, allowing you to map the context to different AWS credentials implementations.  Note that `MtAmazonDynamoDbByAccount` does not support delegation and therefore must always be at the end of the chain when it is used.  See details about chaining below.
 
-```
-MtAmazonDynamoDbByAccount.builder().withAmazonDynamoDbClientBuilder(AmazonDynamoDBClientBuilder.standard())
-                .withAccountCredentialsMapper(new MtAccountCredentialsMapper() {
-                    @Override
-                    public AWSCredentialsProvider getAwsCredentialsProvider(String tenantIdentifier) {
-                        // return an AWSCredentialsProvider based on the tenant identifier
-                    }
-                })
-                .withContext(contextProviderImpl);
+```java
+MtAmazonDynamoDbByAccount.builder()
+    .withAmazonDynamoDbClientBuilder(AmazonDynamoDBClientBuilder.standard())
+    .withAccountCredentialsMapper(
+        new MtAccountCredentialsMapper() {
+            @Override
+            public AWSCredentialsProvider getAwsCredentialsProvider(String tenantIdentifier) {
+                // return an AWSCredentialsProvider based on the tenant identifier
+            }
+        }
+    )
```

Comment:
a right parenthesis (')') on a line by itself seems odd/uncommon—any reason for this choice? (as opposed to, say, removing the whitespace (including a newline) between the right curly brace ('}') before it, additionally (symmetrically) removing the whitespace before `new MtAccountCredentialsMapper() {` (thus putting in on the same line with and immediately after `.withAccountCredentialsMapper(`)?)

Reply:
No strong reason for that. Both the style you suggested and my one are fine for me but, indeed, your one looks more familiar to me. Will update the change as blow:

```java
MtAmazonDynamoDbByAccount.builder()
    .withAmazonDynamoDbClientBuilder(AmazonDynamoDBClientBuilder.standard())
    .withAccountCredentialsMapper(new MtAccountCredentialsMapper() {
        @Override
        public AWSCredentialsProvider getAwsCredentialsProvider(String tenantIdentifier) {
            // return an AWSCredentialsProvider based on the tenant identifier
        }
    })
    .withContext(contextProviderImpl)
    .build();
```

Reply:
Done.